packages/graphback-core/src/runtime/subscriptionFilter.ts

```diff
@@ -0,0 +1,172 @@
+import { convertType, isDateObject } from '../utils/convertType';
```

Comment:
I'll remove `subscription` from the content of this file and from the file name to something like `PredicateBuilder`, `InMemoryFilter` or something along the line. What I see is that we have really nice in memory filter capabilities that can be used for anything other than subscription (I am thinking of in memory datastores ;-)). What's your thoughts on this?

Reply:
For sure, makes total sense!